### PR TITLE
Add logging to our cleanup routine to know what gets deleted when.

### PR DESCRIPTION
**Pull request by markusthoemmes**
Title says it all. We need more info when hunting flakes.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
/test 4.3-e2e-aws-ocp-43

Setup.

**Comment by markusthoemmes** [comment]
/retest

Setup issues... gnaaa

**Comment by markusthoemmes** [comment]
/retest

Setup issues.

File: test/lib.bash [hunk 1]
```diff
@@ -50,22 +50,17 @@ function run_e2e_tests {
     logger.error 'Tests have failures!'
   fi
 
-  wait_for_knative_serving_ingress_ns_deleted || return 1
```
Comment (mgencur) [inline test/lib.bash]: Why are you removing this? Is this intentional?

Comment (markusthoemmes) [reply test/lib.bash]: Yeah, because the namespace should already be removed if the test runs fine.

Comment (markusthoemmes) [reply test/lib.bash]: Actually, going to adjust the test a bit to make sure it's properly gone.

Comment (markusthoemmes) [reply test/lib.bash]: Or just add this line back 😂

Comment (mgencur) [reply test/lib.bash]: I added this as a workaround for Azure. There's a bug which requires patching the kourier ingress. Not sure if the Golang test ensures that.

Comment (mgencur) [reply test/lib.bash]: OK. You added it back:) Thanks

**Comment by markusthoemmes** [comment]
Setup issues

/test 4.4-e2e-aws-ocp-44

**Comment by markusthoemmes** [comment]
/test 4.4-e2e-aws-ocp-44

Flake logged in JIRA.

**Comment by markusthoemmes** [comment]
/test 4.4-e2e-aws-ocp-44

TestMinScale, already being worked on.

**Pull request merged**